### PR TITLE
remix delpoy, transfer

### PR DESCRIPTION
**Pull request by sinev-valentine**

### CODE REVIEW

File: proxy/plugin/erc20_wrapper.py [hunk 1]
```diff
@@ -101,147 +105,247 @@ def create_program_address(seed, program_id):
...
-            raise Exception("Invalid program logs: no result")
-        else:
-            return res[13:]
 
-    def name(self):
-        input = bytearray.fromhex("0306fdde03")
-        (res, log) = self.call(input)
-        if not res.startswith("Program log: "):
-            raise Exception("Invalid program logs: no result")
+def unpack(data):
```
Comment (s-medvedev) [inline proxy/plugin/erc20_wrapper.py]: Why are you copying the code and not including functions from the `eth_proto.py` file instead?
You can use at least `pack`/`unpack`/`trx` this way.

File: proxy/plugin/solana_rest_api.py [hunk 2]
```diff
@@ -351,25 +371,40 @@ def eth_sendRawTransaction(self, rawTrx):
...
+                self.contract_address[eth_signature] = eth_contract_addr
                 return eth_signature
+            else:
+                try:
+                    input = bytearray(trx.callData)
+                    input[0:0] = bytearray.fromhex("03")
+
+                    (contract_sol, contract_nonce) = create_program_address(bytes(trx.toAddress).hex(), evm_loader_id)
+                    (sender_sol, sender_nonce) = create_program_address(sender, evm_loader_id)
+                    (res, log) = call(input, evm_loader_id, contract_sol, sender_sol, self.signer, self.client)
```
Comment (s-medvedev) [inline proxy/plugin/solana_rest_api.py:392]: At this point, you do not need to wait for the transaction to complete. This call only sends the transaction to the chain, and the metamask then performs special requests to make sure that the transaction has been applied. In this PR, let's leave it as done, and when you rewrite the entire transaction to send, you will correct the logic of the work.

File: proxy/plugin/erc20_wrapper.py (hunk 1 above)
Comment (sinev-valentine) [reply proxy/plugin/erc20_wrapper.py]: fixed

File: proxy/plugin/solana_rest_api.py (hunk 2 above)
Comment (sinev-valentine) [reply proxy/plugin/solana_rest_api.py:392]: no changes

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**